src/fsverity/mod.rs

```diff
@@ -142,21 +142,204 @@ pub fn ensure_verity_equal(
 
 #[cfg(test)]
 mod tests {
-    use crate::fsverity::Sha256HashValue;
```

Comment:
The removal of use Sha256HashValue should have been handled in the previous test cleanups commit.